images/origin/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN ln -s /usr/bin/openshift /usr/bin/oc && \
     ln -s /usr/bin/openshift /usr/bin/openshift-deploy && \
     ln -s /usr/bin/openshift /usr/bin/openshift-docker-build && \
     ln -s /usr/bin/openshift /usr/bin/openshift-sti-build && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-router && \
+    ln -s /usr/bin/openshift /usr/bin/openshift-f5-router && \
```

Comment:
Ok - see why you would need it if you removed it from the f5 dockerfile. imo, it feels better to remove it here and just keep it in the f5 dockerfile - its consistent with the haproxy one.